package.json

```diff
@@ -48,13 +49,14 @@
     "request": "2.65.0",
     "scrypt-hash": "1.1.13",
     "through": "2.3.8",
-    "uuid": "1.4.1"
+    "uuid": "1.4.1",
+    "web-push": "git+https://github.com/eoger/web-push.git#node-0.10"
```

Comment:
Do you think you'll be able to get your changes accepted upstream, or will we need to deploy from this fork on an ongoing basis?